.circleci/config.yml

```
@@ -0,0 +1,75 @@
+                headers = {'X-Api-Key': 'f31f2871d328a52a45fefadc09a1c67d0dd5d53d'}
+                response = requests.get(url, headers=headers)
+                if response.status_code == 200:
+                    return response.json()
+                else:
+                    raise Exception("Erro ao buscar logs: " + response.text)
+
+            logs = fetch_nextdns_logs()
+            
+            timestamp = datetime.now().strftime("%Y%m%d_%H%M%S")
+            filename = f'nextdns_logs_{timestamp}.json'
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)